src/ssl.c

```diff
@@ -3438,26 +3441,29 @@ static int wolfssl_encrypt_buffer_key(byte* der, word32 derSz, byte* password,
         return SSL_FATAL_ERROR;
     }
 #else
-    (void) passwordSz;
+
```

Comment:
Not sure that this else case is needed any longer with moving (void)passwordSz higher up.